src/map/utils/fishingutils.cpp

```diff
@@ -76,6 +84,28 @@ uint16 GetMessageOffset(uint16 ZoneID)
 	return MessageOffset[ZoneID];
 }
 
+// get player skill with item mod bonus added
+uint16 getPlayerFishingSkill(CCharEntity* PChar)
+{
+    uint16 skill = 0;
+
+    // player skill
+    skill += PChar->RealSkills.skill[SKILL_FISHING];
+
+#ifdef _DSP_SYNTH_DEBUG_MESSAGES_
+    ShowDebug(CL_CYAN"player skill = %u\n" CL_RESET, skill);
```

Comment:
"Player fishing skill" (so we know what skill you're talking about when reading the log file)

Reply:
Done